spec/models/vaccination_record_spec.rb

```diff
@@ -39,25 +39,50 @@
 #
 require "rails_helper"
 
-describe VaccinationRecord do
-  it "validates that the vaccine and the batch vaccines match" do
-    patient_session = create(:patient_session)
-    vaccine = patient_session.campaign.vaccines.first
-    different_vaccine = create(:vaccine)
-    batch = create(:batch, vaccine: different_vaccine)
+describe VaccinationRecord, type: :model do
+  subject(:vaccination_record) do
+    create(
+      :vaccination_record,
+      patient_session:
+        create(:patient_session, session_attributes: { campaign: })
+    )
+  end
+
+  let(:campaign) do
+    create(
+      :campaign,
+      :active,
+      academic_year: 2020,
+      start_date: Date.new(2020, 1, 1),
+      end_date: Date.new(2020, 12, 31)
+    )
+  end
 
-    subject =
-      build(
-        :vaccination_record,
-        administered: true,
-        vaccine:,
-        batch:,
-        patient_session:
+  describe "validations" do
+    it do
```

Comment:
Can we give this spec a description? I think it makes sense to skip a description if the spec all fits on a line, but this one spans a few lines.

Reply:
Yeah no problem, I've added one.